### ISSUE TITLE
--timeout ignored in mochapack.opts

### ISSUE THREAD

**Issue by Spacexplosion** [sysgears/mochapack#104]
Appending --timeout to the CLI args changes default timeout behavior, but putting the same argument in the mochapack.opts file does not.

Versions:
    "mocha": "^7.2.0",
    "mochapack": "^2.1.2",
    "webpack": "^5.38.1",

**Comment by Spacexplosion** [comment]
Actually, it seems everything in mochapack.opts is ignored. Specifying `--opts mochapack.opts` doesn't help

**Comment by AnthonyElcocks** [comment]
This still seems to be an issue. Is it on the radar?

**Comment by kaorukobo** [comment]
It seems there are recent changes that breaks around CLI options parser.

## How to Reproduce

```sh
# Set up mochapack v2.1.2 project.
rm -rf sample-project || true
mkdir -p sample-project
cd sample-project
yarn add -D mochapack mocha@^8 webpack@^4

# 1. --help does not work.
yarn exec mochapack --help

# 2. --webpack-env value is not passed to webpack.config.js.
cat > webpack.config.js <<EOS
module.exports = (env) => {
    if (env === undefined) {
        throw 'env is undefined!'
    }
    return {}
}
EOS
yarn exec mochapack --webpack-env=production
```

**Comment by jcjp** [comment]
> Actually, it seems everything in mochapack.opts is ignored. Specifying `--opts mochapack.opts` doesn't help

Yes correct more information is on this bug: https://github.com/sysgears/mochapack/issues/73. I wish that mochapack team would utilize the `.mocharc.js` file or any other alternative for now added all my options on my command.
